devops/docker/ansible/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.5-slim
+
+ENV ADMIN_PASSWORD=$ADMIN_PASSWORD
+ENV ESIMMON_PASSWORD=$ESIMMON_PASSWORD
```

Comment:
Do we need to build these into the container? Its probably not a good idea.

Reply:
No we don't, removed